allauth/socialaccount/providers/authentiq/views.py

```diff
@@ -0,0 +1,37 @@
+import requests
+
+from allauth.compat import urljoin
+from allauth.socialaccount import app_settings
+from allauth.socialaccount.providers.oauth2.views import (
+    OAuth2Adapter,
+    OAuth2CallbackView,
+    OAuth2LoginView,
+)
+
+from .provider import AuthentiqProvider
+
+
+class AuthentiqOAuth2Adapter(OAuth2Adapter):
+    provider_id = AuthentiqProvider.id
+    default_provider_url = 'https://connect.authentiq.io/'
+
+    settings = app_settings.PROVIDERS.get(provider_id, {})
+
+    provider_url = settings.get('PROVIDER_URL', default_provider_url)
+
+    access_token_url = urljoin(provider_url + '/', 'token')
+    authorize_url = urljoin(provider_url + '/', 'authorize')
+    profile_url = urljoin(provider_url + '/', 'userinfo')
```

Comment:
Leave it like this, optionally with a comment referring to that people may have specified a path and forgot the trailing /